### PR TITLE
Only check for an autosave file the first time the window is shown

### PR DESCRIPTION
**Pull request by mathturtle**
@cquammen @Hovden For #515.

### CODE REVIEW

File: tomviz/MainWindow.cxx [hunk 1]
```diff
@@ -439,7 +440,11 @@ void MainWindow::moduleChanged(Module*)
 void MainWindow::showEvent(QShowEvent *e)
 {
   Superclass::showEvent(e);
-  QTimer::singleShot(1,this,SLOT(checkForAutosaveFile()));
+  if (this->Internals->isFirstShow)
+  {
+    this->Internals->isFirstShow = false;
+    QTimer::singleShot(1,this,SLOT(checkForAutosaveFile()));
```
Comment (cryos) [inline tomviz/MainWindow.cxx]: Can we add spaces after commas, makes it easier to glance over

**Comment by cryos** [comment]
Looks good to me

**Comment by mathturtle** [comment]
Added a commit to fix the spacing.

**Comment by cryos** [comment]
Looks great, thanks!

**Pull request merged**